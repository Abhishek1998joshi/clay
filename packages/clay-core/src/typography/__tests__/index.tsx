/**
 * SPDX-FileCopyrightText: Â© 2022 Liferay, Inc. <https://liferay.com>
 * SPDX-License-Identifier: BSD-3-Clause
 */

 import {Heading} from '..';
 import {cleanup, render} from '@testing-library/react';
 import React from 'react';

 describe('Heading', () => {
	afterEach(cleanup);

	it('renders as h1 with normal style', () => {
		

		render(
			<Heading
		    level={1}
		    weight='normal'
	        >
		        This is a Heading
	        </Heading>
		);
        
        expect(document.querySelector('.font-weight-normal')).toBeTruthy();
		expect(document.body).toMatchSnapshot();
	});

    it('renders as h3 with semi-bold style', () => {
		

		render(
			<Heading
		    level={3}
		    weight='semi-bold'
	        >
		        This is a Heading
	        </Heading>
		);
        
        expect(document.querySelector('.font-weight-semi-bold')).toBeTruthy();
		expect(document.body).toMatchSnapshot();
	});
});