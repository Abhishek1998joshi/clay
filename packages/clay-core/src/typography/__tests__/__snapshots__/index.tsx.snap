// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading renders as h1 with normal style 1`] = `
<body>
  <div>
    <h1
      class="font-weight-normal"
    >
      This is a Heading
    </h1>
  </div>
</body>
`;

exports[`Heading renders as h3 with semi-bold style 1`] = `
<body>
  <div>
    <h3
      class="font-weight-semi-bold"
    >
      This is a Heading
    </h3>
  </div>
</body>
`;
